sway-book/src/documentation/language/program-types/script.md

```diff
@@ -0,0 +1,19 @@
+# Scripts
+
+A script is an executable that does not need to be deployed because it only exists during a transaction.
+
+It can be used to replicate the functionality of contracts, such as routers, without the cost of deployment or increase of the blockchain size.
+
+Some properties of a script include:
+
+- It cannot be called by a contract
+- It is stateless but can interact with storage through a contract
```

Comment:
Scripts can impact the state but the scripts themselves aren't stored on the chain, perhaps a better way to say this than `stateless`?